aip_xx1_gen2_launch/launch/lidar.launch.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2024 Tier IV, Inc. All rights reserved.
```

Comment:
```suggestion
# Copyright 2024 TIER IV, Inc. All rights reserved.
```
:police_officer:

Reply:
I fixed the copyright description in [d75f529](https://github.com/tier4/aip_launcher/pull/326/commits/d75f52983ad62b35cd70182e4bc6d589ffede84d)